lesson03/assignment/index.js

```diff
@@ -0,0 +1,145 @@
+    ]
+  }
+ ]
+=======
+const cardJSON = `[
+    {
+      "name": "Paolo Maldini",
+      "jobTitle": "Front End Developer",
+      "company": "Web Weavers",
+      "experience": "3 years",
+      "school": "University of Washington",
+      "major": "Marketing",
+      "email": "paolo@example.com",
+      "linkedInUrl": "paolo.linkedinprofile.com",
+      "codeLanguages": [
+          "HTML", "CSS", "JavaScript", "Node", "Express"
+      ]
+    },
+    {
+      "name": "Barbara Bonansea",
+      "jobTitle": "Software Engineer",
+      "company": "Excellence in the Cloud",
+      "experience": "12 years",
+      "school": "University of Southern California",
+      "major": "Computer Science",
+      "email": "barbara@example.com",
+      "linkedInUrl": "barbara.linkedinprofile.com",
+      "codeLanguages": [
+          "HTML", "JavaScript", "C", "Go"
+      ]
+    },
+    {
+      "name": "Javier Hernandez",
+      "jobTitle": "User Experience Engineer",
+      "company": "Web Sites and More",
+      "experience": "5 years",
+      "school": "Seattle University",
+      "major": "Performing Arts",
+      "email": "javier@example.com",
+      "linkedInUrl": "javier.linkedinprofile.com",
+      "codeLanguages": [
+          "HTML", "CSS"
+      ]
+    },
+    {
+      "name": "Maribel Dominguez",
+      "jobTitle": "Front End Engineer",
+      "company": "Bits and Bytes",
+      "experience": "6 years",
+      "school": "University of Washington",
+      "major": "Mechanical Engineering",
+      "email": "maribel@example.com",
+      "linkedInUrl": "maribel.linkedinprofile.com",
+      "codeLanguages": [
+         "HTML", "CSS", "JavaScript", "React", "Vue", "Redux"
+      ]
+    }
+   ]
+  
+  `
+
+const card = JSON.parse(cardJSON)
+
+
+// Loop through all elements of the list
+let output = card.map(function(c){
+    return `
+    <section class="card">
+            <section class="title">
+            <img src="./img/headshot.jpg" alt="portrait of Paolo Maldini" class="titleimg">
+            <h1>${c.name}</h1>
+            <h2>${c.jobTitle}</h2>
+        </section>
+        <section class="content">
+            <ul>
+            <li><span class="bold">${Object.keys(c)[2]}:</span> ${Object.values(c)[2]}</li>
+            <li><span class="bold">${Object.keys(c)[3]}:</span> ${Object.values(c)[3]}</li>
+            <li><span class="bold">${Object.keys(c)[4]}:</span> ${Object.values(c)[4]}</li>
+            <li><span class="bold">${Object.keys(c)[5]}:</span> ${Object.values(c)[5]}</li>
+            <li><span class="bold">${Object.keys(c)[6]}:</span> ${Object.values(c)[6]}</li>
+            <li><span class="bold">${Object.keys(c)[8]}:</span> ${Object.values(c)[8]}</li>
+        </ul>
+        <div class="link">
+        <img src="./img/linkedin.svg" alt="linked in icon">
+        <p>${c.linkedInUrl}</p>
+        </div>
+        </section>
+        </section>`;
+
+});
+document.querySelector("main").innerHTML = output
+  
+>>>>>>> Stashed changes
```

Comment:
Looks like you had a 'merge conflict', you'll want to resolve these. Essentially, delete the lines with `>>>>>` or `<<<<<` and selectively pick the code you wish to keep

Reply:
Some more info on merge conflicts: https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts

Kind of a tricky concept. Definitely a good one to bring up in office hours and can talk through this more